analaysis/plant/plant_functional_types/plant_functional_type_species_classification_base.R

```diff
@@ -0,0 +1,182 @@
+#' ---
+#' title: Plant functional type species classification base
+#'
+#' description: |
+#'     This script classifies species into plant functional types (PFTs).
+#'     At the moment, these PFTs only contain trees.
+#'     This classification is predominantly based on the one provided by
+#'     Kohler and Huth (1998; DOI https://doi.org/10.1016/S0304-3800(98)00066-0)
+#'     and contains the following PFTs:
+#'     -emergent trees
+#'     -overstory trees
+#'     -understory trees
+#'     -pioneer trees
+#'     For the understory PFT, the species list is expanded by also including
+#'     the species listed in group 9 (small shade trees) in
+#'     Phillips et al. (2000; DOI https://doi.org/10.1016/S0378-1127(00)00666-6).
+#'     This combined PFT species classification is then applied to the
+#'     SAFE census dataset.
+#'     The output of this script generates a CSV file containing a list of
+#'     species and their respective PFT.
+#'     This CSV file can then be loaded when working with other datasets
+#'     (particularly those related to updating T model parameters).
+#'     In a follow up script, the remaining species that have not been assigned
+#'     a PFT yet will be assigned into one based on their species maximum height
+#'     relative to the PFT maximum height.
+#'
+#' VE_module: Plant
+#'
+#' author:
+#'   - name: Arne Scheire
+#'
+#' status: final
+#'
+#'
+#' input_files:
+#'   - name: tree_census_11_20
+#'     path: ve_data_science\data\primary\plant\tree_census
```

Comment:
These paths (here and below) should start with `data` and use forward slashes: `data/primary/plant/tree_census`.

Or at least, that's what I think we'll want to use once the validation is all hooked up!